src/components/ui/BulkSellModal.tsx

```
@@ -39,6 +39,21 @@ export const BulkSellModal: React.FC<BulkSellProps> = ({
     customAmount.greaterThan(itemAmount) || customAmount.lessThanOrEqualTo(0);
 
+  const minValid = setPrecision(
+    new Decimal(1).div(10 ** maxDecimalPlaces),
+    maxDecimalPlaces,
+  );
+
+  const safeHalf = Decimal.min(
+    Decimal.max(
+      setPrecision(itemAmount.mul(0.5), maxDecimalPlaces),
+      minValid,
+    ),
```

ESLint

[error] Replace `⏎······setPrecision(itemAmount.mul(0.5),·maxDecimalPlaces),⏎······minValid,⏎····` with `setPrecision(itemAmount.mul(0.5),·maxDecimalPlaces),·minValid`

(prettier/prettier)